unfetter-discover/unfetter-ui
# Update assessment summary page to use new risk-to-attack pattern model & service changes.

supports unfetter-discover/unfetter#1084

REQUIRES PR: https://github.com/unfetter-discover/unfetter-store/pull/221

To Test:

- Insure that above PR branch is merged to dev or checked out and running
- Test assessment summary page to see if it loads properly for all types of assessments
- Capabilities types should now populate the "Where are you most vulnerable" chart with data.
- ASSESSED ITEM links will be bad because no stix for capabilities; will disable that in a future iteration.
